### ISSUE TITLE
does cmb2 work with polylang?

### ISSUE THREAD

**Issue by sandrodz** [CMB2/CMB2#966]
Lets say I'm using theme options snippet to create site options page. How would I make normal text input translatable?

**Comment by tw2113** [comment]
Honestly a challenge that is faced by anyone setting up this type of thing, CMB2-powered or not. Best solution I can think of, that isn't specific to CMB2, is some sort of language switch that renders the page for that specific language, and saves the value to an array or something else, keyed by the current language code. However, this is just a stream of ideas, and I don't have any code at all for it.

**Comment by sandrodz** [comment]
I've got it working with polylang strings. Its not perfect, but works.

![screen shot 2017-05-05 at 12 25 21 pm](https://cloud.githubusercontent.com/assets/8479569/25738403/f6a53428-318d-11e7-8768-928e68eacd41.png)
![screen shot 2017-05-05 at 12 25 25 pm](https://cloud.githubusercontent.com/assets/8479569/25738404/f6ad7b7e-318d-11e7-8d17-f5c9ebeeed96.png)

I looked at polylang wpseo integration and took example from there.
